tools/static-assets/skel-full/package.json

```diff
@@ -5,6 +5,7 @@
     "start": "meteor run"
   },
   "dependencies": {
-    "meteor-node-stubs": "~0.2.0"
+    "babel-runtime": "^6.18.0",
```

Comment:
I can see your concern about the `dependencies` here, but keeping the actual latest version number here isn't critical.

The most recent compatible version of the packages will still be installed anyhow (thanks to semver) when the user runs `meteor npm install`.

And for those who don't run `meteor npm install`, the version listed in their own (newly created) `package.json` will get updated soon after they've created their app since users will be given the warning to `meteor npm install babel-runtime --save`.  Running that will bump this version, and install the latest simultaneously.

They can probably just be updated when someone notices.